include/crm/msg_xml_compat.h

```diff
@@ -880,6 +880,9 @@ extern "C" {
 //! \deprecated Do not use
 #define XML_ACL_TAG_ROLE_REFv1 "role_ref"
 
+//! \deprecated Do not use
+#define XML_ACL_TAG_READ "read"
```

Comment:
This does have a replacement, but it's not exact -- the old name incorrectly referred to a tag, while the tag was deprecated in 1.1.12. The new symbol correctly refers to a value.